### DISCUSSION TITLE
nao consigo instalar o opencv no bcnc

### DISCUSSION THREAD

**Discussion by Samlokihel** [vlachoudis/bCNC#1651] [General]
alguem poderia me ajudar a instalar o opencv no meu bcnc uso python 2.7.......

**Comment by JipeB** [comment]
Qual sistema operacional você está usando? Windows, Linux ...

**Reply by Samlokihel** [reply]
Hello, I'm using 7 ultimate 64 bit windons,
python 2.7.15,
bcnc 0.9.14. Thanks for the answer.

**Comment by JipeB** [comment]
Have you tried the command line installation?
python.exe -m pip install bCNC

https://github.com/vlachoudis/bCNC/wiki/Installation

**Reply by Samlokihel** [reply]
OBRIGADO DEU CERTO...AGORA APRENDERA CONFIGURAR A CAMERA E USAR O DETECTOR DE BORDAS. OBRIGADO A VOCE E A TODOS QUE AJUDARAM
